asyncpgsa/pool.py

```diff
@@ -5,10 +5,16 @@
 
 
 class SAPool(Pool):
+    __slots__ = ('_debug',)
```

Comment:
I may be misreading the docs, but AFAIK they only apply to the class they define, and extended classes will be a `__dict__` unless they add additional `__slots__`:

> The action of a __slots__ declaration is limited to the class where it is defined. As a result, subclasses will have a __dict__ unless they also define __slots__ (which must only contain names of any additional slots).

(Bullet 6 from [here](https://docs.python.org/2/reference/datamodel.html#slots))